app/bg/dbs/settings.js

```diff
@@ -93,7 +101,11 @@ export const get = function (key) {
   // stored values
   return setupPromise.then(() => cbPromise(cb => {
     db.get(`SELECT value FROM settings WHERE key = ?`, [key], (err, row) => {
-      if (row) { row = row.value }
+      if (row) {
+        row = row.value
+        if (['default_search_engines', 'selected_search_engine'].includes(key))
+          row = JSON.parse(row)
```

Comment:
We should wrap the `parse()` calls in a try/catch and then fall back to the default value on failure